frontend/src/components/DashboardTable.vue

```diff
@@ -183,43 +184,61 @@ export default {
+        this.loadingTable = true;
+        this.requests = [];
+        for(const schoolId of this.userInfo.userSchoolIDs) {
 
-      ApiService.apiAxios.get(ApiRoutes.school.SCHOOL_DETAILS_BY_ID + `/${this.userInfo.activeInstituteIdentifier}`).then(response => {
+          ApiService.apiAxios.get(ApiRoutes.school.SCHOOL_DETAILS_BY_ID + `/${schoolId}`).then(response => {
 
-        let rawDate = response.data.updateDate === null ? response.data.openedDate : response.data.updateDate;
-        this.schoolsLastUpdateDate = new Date(rawDate).toISOString().slice(0,10).replace(/-/g,'/');
-        this.getSchoolContactsLastUpdate(response.data);
-      }).catch(error => {
-        //to do add the alert framework for error or success
-        console.error(error);
-      }).finally(() => {
-        this.loadingTable = false;
-      });
+            let rawDate = response.data.updateDate === null ? response.data.openedDate : response.data.updateDate;
+            let thisSchoolsLastUpdateDate = new Date(rawDate).toISOString().slice(0, 10).replace(/-/g, '/');
+
+            if(thisSchoolsLastUpdateDate !== null) {
+              if (thisSchoolsLastUpdateDate > this.schoolsLastUpdateDate) {
+                this.schoolsLastUpdateDate = thisSchoolsLastUpdateDate;
+              }
+            }
+          }).catch(error => {
+            //to do add the alert framework for error or success
+            console.error(error);
+          }).finally(() => {
+            this.loadingTable = false;
+          });
+        }
+      }
     },
     redirectToSchools(){
       router.push('/schools');
     },
     redirectToDistrictDetails(){
       router.push('/district/' + this.userInfo.activeInstituteIdentifier);
     },
-    getSchoolContactsLastUpdate(school){
-      this.schoolContactsLastUpdateDate = '';
-      let lastUpdate = '';
+    getSchoolContactsLastUpdate(){
+      if(this.userInfo.activeInstituteType === 'SCHOOL') {
+        ApiService.apiAxios.get(ApiRoutes.school.SCHOOL_DETAILS_BY_ID + `/${this.userInfo.activeInstituteIdentifier}`).then(response => {
 
-      for (const contact of school.contacts){
-        if(contact.updateDate !== null) {
-          if (contact.updateDate > lastUpdate) {
-            lastUpdate = contact.updateDate;
+          for (const contact of response.data.contacts) {
+            let rawDate = contact.updateDate === null ? contact.effectiveDate : contact.updateDate;
+            let thisContactLastUpdated = new Date(rawDate).toISOString().slice(0, 10).replace(/-/g, '/');
+
+            if (thisContactLastUpdated !== null) {
+              if (thisContactLastUpdated > this.schoolContactsLastUpdateDate) {
+                this.schoolContactsLastUpdateDate = thisContactLastUpdated;
+              }
+            }
           }
-        } else {
-          lastUpdate = contact.effectiveDate;
-        }
+        }).catch(error => {
+          //to do add the alert framework for error or success
```

Comment:
There's an alert feature available
https://github.com/bcgov/EDUC-EDX/blob/master/frontend/src/mixins/alertMixin.js

refer to this for implementation
https://github.com/bcgov/EDUC-EDX/blob/master/frontend/src/components/SecureExchange/MessageDisplay.vue#:~:text=import%20alertMixin%20from%20%27%40/mixins/alertMixin%27%3B